src/backend/distributed/executor/multi_utility.c

```diff
@@ -2142,8 +2203,15 @@ ErrorIfDistributedAlterSeqOwnedBy(AlterSeqStmt *alterSeqStmt)
 		return;
 	}
 
-	/* see whether the sequences is already owned by a distributed table */
-	if (sequenceIsOwned(sequenceId, &ownedByTableId, &ownedByColumnId))
+#if (PG_VERSION_NUM >= 100000)
+	sequenceOwned = sequenceIsOwned(sequenceId, DEPENDENCY_AUTO, &ownedByTableId,
```

Comment:
Ping? Afaics this hasn't yet been addressed?  At the least it seems we need a test checking that identity type columns are handled without unexpected errors (i.e. by erroring out while distributing such tables instead).